spec/factories/state_file_nj_intakes.rb

```diff
@@ -15,6 +16,7 @@
 #  current_sign_in_ip                                     :inet
 #  current_step                                           :string
 #  date_electronic_withdrawal                             :date
+#  df_data_import_failed_at                               :datetime
```

Comment:
I think you need to extend the ignored columns from the base intakes to get this removed!